gcp-repo/metadata.p2.inf

```diff
@@ -27,7 +27,7 @@ units.1.properties.5.name=org.eclipse.equinox.p2.doc.url
 units.1.properties.5.value=https://cloud.google.com/eclipse/docs/
 units.1.provides.1.namespace=org.eclipse.equinox.p2.iu
 units.1.provides.1.name=com.google.cloud.tools.eclipse.category
-units.1.provides.1.version=1.0.0
+units.1.provides.1.version=1.0.1
```

Comment:
The `units.1.version` determines what's shown the user.
```diff
diff --git a/gcp-repo/metadata.p2.inf b/gcp-repo/metadata.p2.inf
index 010b3c0..c234b59 100644
--- a/gcp-repo/metadata.p2.inf
+++ b/gcp-repo/metadata.p2.inf
@@ -14,7 +14,7 @@ requires.1.range=[1.0.0,1.0.1]
 requires.1.greedy=true
 
 units.1.id=com.google.cloud.tools.eclipse.category
-units.1.version=1.0.1
+units.1.version=999
 units.1.properties.1.name=org.eclipse.equinox.p2.type.category
 units.1.properties.1.value=true
 units.1.properties.2.name=org.eclipse.equinox.p2.name
```
produces:
<img width="693" alt="screen shot 2017-03-09 at 11 12 48 pm" src="https://cloud.githubusercontent.com/assets/202851/23781851/133260ee-051e-11e7-8e9e-417e9b1b006c.PNG">